airflow/www/views.py

```diff
@@ -751,22 +755,27 @@ def log(self):
                     ti_execution_completed or ti_ran_more_than_once)
 
                 # S3
-                if remote_log_path.startswith('s3:/'):
-                    remote_log += log_utils.S3Log().read(
-                        remote_log_path, return_error=surface_log_retrieval_errors)
+                if remote_log_dir.startswith('s3:/'):
+                    for i in xrange(len(remote_logs)):
```

Comment:
Nit: A little bit cleaner is 
for remote_log, i in enumerate(remote_logs): (and then use remote_log instead of remote_logs[i] below)